src/03-kinematics.sql

```diff
@@ -0,0 +1,26 @@
+\connect wallabydb
+
+CREATE TABLE wallaby.kinematic_model (
+  "source_name" varchar PRIMARY KEY,
```

Comment:
This table should also have a id field for its primary key right? 
`"id" BIGSERIAL PRIMARY KEY`

Reply:
I thinks so. We can wait @venkat to confirm it.

Reply:
Solved in the latest commits. Thanks !